caffeine/src/test/java/com/github/benmanes/caffeine/cache/AsyncAsMapTest.java

```diff
@@ -2594,7 +2595,7 @@ public void entrySet_remove_null(AsyncCache<Int, Int> cache, CacheContext contex
   @Test(dataProvider = "caches")
   public void entrySet_remove_nullKey(AsyncCache<Int, Int> cache, CacheContext context) {
     var future = Iterables.getFirst(cache.asMap().values(), context.absentValue().toFuture());
-    assertThat(cache.asMap().entrySet().remove(Maps.immutableEntry(null, future))).isFalse();
+    assertThat(cache.asMap().entrySet().remove(Maps.<@Nullable Int, CompletableFuture<Int>>immutableEntry(null, future))).isFalse();
```

Comment:
These explicit type arguments are needed due to limitation in NullAway's inference.  They kind of clutter the code, so we could just suppress instead

Reply:
either is fine with me

Reply:
Ok, let me change to suppress.  When we improve NullAway's inference we then should be able to detect the unnecessary suppression and remove it